frida_mode/src/instrument/instrument_coverage.c

```diff
@@ -829,7 +832,7 @@ void instrument_coverage_unstable_find_output(void) {
     if (link == NULL) { FFATAL("Failed to read link: %s", fullname); }
 
     gchar *basename = g_path_get_basename(link);
-    if (g_strcmp0(basename, "default") != 0) {
+    if (g_strcmp0(basename, instance_name) != 0) {
```

Comment:
instance_name is “out/foobar” not foobar

Reply:
No, [gchar *instance_name = g_path_get_basename(path_tmp);](https://github.com/AFLplusplus/AFLplusplus/pull/2169/files#diff-8a67a75c5241f69cb74a414ec4ff56cab32dcead4104b3d7574638550df87a64R822) takes “out/foobar” from getenv("AFL_CUSTOM_INFO_OUT"); and returns "foobar"